graspologic/embed/base.py

```diff
@@ -4,7 +4,9 @@
 from abc import abstractmethod
 
 import numpy as np
+from scipy.sparse.linalg import LinearOperator
```

Comment:
Why are you importing LinearOperator to this file, I don't see it used here?

Reply:
Same with preprocessing

Reply:
good catch, was using it in an older implementation, 8c8d974